src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -117,28 +119,19 @@ public Launcher decorate(final Launcher launcher, final Node node) {
             public Proc launch(ProcStarter starter) throws IOException {
                 boolean quiet = starter.quiet();
                 FilePath pwd = starter.pwd();
-
-                String [] cmdEnvs = starter.envs();
-
-                //check if the cmd is sourced from Jenkins, rather than another plugin; if so, skip cmdEnvs as we are getting other environment variables
-                for (String cmd : cmdEnvs) {
-                    if (cmd.startsWith(JENKINS_HOME)) {
```

Comment:
this is no longer needed ? 
it's being reimplemented in 
https://github.com/jenkinsci/kubernetes-plugin/pull/245/files#diff-8a1ab34452b01252e89ffeee0b2eda45R201